src/main/java/com/eleks/academy/whoami/model/response/PlayerWithState.java

```diff
@@ -19,4 +19,9 @@ public class PlayerWithState {
 
 	private PlayerState state;
 
+	public PlayerWithState(SynchronousPlayer player) {
+		this.player = player;
+		this.state = PlayerState.READY;
```

Comment:
suggest you not hardcode, but add state as a parameter to the constructor and in a place where you create an instance put `PlayerState.READY` as a value

Reply:
Resolved